src/lib/utils/accounts.ts

```diff
@@ -29,3 +33,43 @@ export const isValidAccountNumber = (
   if (accountNumber === null) return false
   return Number.isInteger(accountNumber) && accountNumber >= 0
 }
+
+export const isUtxoAccount = (account: Account<KnownChainIds>): account is Account<UtxoChainId> => {
+  return isUtxoChainId(account?.chainId)
+}
```

Comment:
q: do we need this vs. `isUtxoAccountId`? Only usage is in `findUtxoAccountIdByAddress` below and we could `isUtxoAccountId(accountId)` the conventional way instead - probably fine to just cast if that's the rationale

Not super strict on this one, fine to keep but let's remove the export keyword at least